### ISSUE TITLE
Can't you upload the db.json?

### ISSUE THREAD

**Issue by robertveloso** [stjepangolemac/neural-captcha#1]

**Comment by robertveloso** [comment]
I have 9k labeled captchas like this one, but I don't know how to generate the db.json
I made one that works with this captcha but in python using keras and tensorflow/theano. I needed an standalone exe file and python generate big exe so I find this project. Could you help me?

**Comment by stjepangolemac** [comment]
I too used this type of captcha to make a character classifier in python/tensorflow. Can 50k of extracted, classified and processed characters be of any use to you @RobertVeloso?

When it comes to creating standalone programs from models I have no experience but I suppose they can be hosted in the cloud or locally.

**Comment by robertveloso** [comment]
Yes, of course! If you could please send me those files or the complete nodejs project. It would really help me to create the standalone exe file. Maybe in dropbox or anything you find easy.

**Comment by stjepangolemac** [comment]
Here it is, 50k of extracted and correctly classified characters in numpy binary format.

[DATASET](https://we.tl/eaQXzpBhxp)

Extract it, and then load it in Python like this:
``` python
import numpy as np
data = np.load("part.0.npz")

letters = data["letters"]
matrices = data["matrices"]
```

Both correct letters and their appropriate images (converted into 2d float matrices) are numpy arrays of 5000 in length in each of 10 parts.

I am planning to upload 1.5M characters and make it a public dataset, so stay tuned.

**Comment by robertveloso** [comment]
I will, and thank you so much, I'll try to make the best use of it. Since Where I Live we don't have much access to internet in my country. And I plan to make a program to boost our time from all of the captcha we have to provide in my school. So thank you :)

**Comment by robertveloso** [comment]
If you could also upload the db.json file, because I might do an server using node so people would make the request to solve this captcha. That would be awesome

**Comment by stjepangolemac** [comment]
I am afraid I lost it in the process of replacing my laptop a couple of months ago. You can also convert that from numpy to json.

**Comment by robertveloso** [comment]
Yes, that's true! I will try to convert it today or maybe tomorrow. Thank you
